open-apparel-registry/open-apparel-registry
# Change the official name of Macedonia

## Overview

In March of 2019 the country changed its name from The Former Yugoslav Republic
of Macedonia to North Macedonia. This commit updates our country code parsing
and model choice fields to match.

https://en.wikipedia.org/wiki/ISO_3166-2:MK
Connects #510 

## Demo

<img width="560" alt="Screen Shot 2019-05-17 at 8 08 45 AM" src="https://user-images.githubusercontent.com/17363/57937566-06171f80-787b-11e9-8f69-92751044d9f5.png">


## Testing Instructions

* Run `./scripts/update` to apply the migration
* Log in and upload
[macedonia.csv.txt](https://github.com/open-apparel-registry/open-apparel-registry/files/3192078/macedonia.csv.txt)
* Run `./scripts/manage batch_process --list-id {the-id} --action parse` and verify that all three rows are correctly parsed as North Macedonia
* Verify that the country drop down on the main page contains North Macedonia


## Checklist
